test/compact.jl

```diff
@@ -148,6 +139,28 @@ end
     @test similar_strings(get_model_string(model2), expected_string)
   end
 
+#=  # This test is broken:
```

Comment:
Not sure what this commented block means? It looks equivalent to the Hierarchical models test above, except with a syntax error in the construction of `model1`.

Reply:
I think this is just a git mistake, sorry. (Was broken for a bit locally.)

Reply:
Removed